homeassistant/components/fastdotcom/__init__.py

```diff
@@ -35,15 +40,15 @@
 )
 
 
-async def async_setup(hass, config):
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     """Set up the Fast.com component."""
     conf = config[DOMAIN]
     data = hass.data[DOMAIN] = SpeedtestData(hass)
 
     if not conf[CONF_MANUAL]:
         async_track_time_interval(hass, data.update, conf[CONF_SCAN_INTERVAL])
 
-    def update(call=None):
+    def update(call: bool | None = None) -> None:
```

Comment:
```suggestion
    def update(service_call: ServiceCall) -> None:
```

`from homeassistant.core import ServiceCall`